hw2/bor/Tree.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+
+namespace bor
+{
+    class Tree
+    {
+        private Node root;
+        public int Size;
+        public Tree()
+        {
+            root = new Node();
+            this.Size = 0;
+        }
+        public bool Add(string element)
```

Comment:
```suggestion
        }
        
        public bool Add(string element)
```
Все методы всегда разделяются пустой строкой